crates/ruma-client-api/src/media/get_content_as_filename.rs

```diff
@@ -91,7 +91,7 @@ pub mod v3 {
         ///
         /// [MDN]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#Syntax
         #[ruma_api(header = CONTENT_DISPOSITION)]
-        pub content_disposition: Option<String>,
+        pub content_disposition: String,
```

Comment:
This is bad for backwards-compatibility on the client side, I would leave it optional here but required in the constructor. And maybe say in the docs that it is required for homeservers.

Note that the content type too becomes required.

Reply:
> This is bad for backwards-compatibility on the client side

Ah, I didn't consider that, nice catch.